### ISSUE TITLE
[Linux] Header Editor cuts off binary header display

### ISSUE THREAD

**Issue by Sanaki** [SourMesen/Mesen#801]
Under mono, the binary header display cuts off the last five bytes. See image below.
![mesenheader](https://user-images.githubusercontent.com/1374333/78703017-09832180-78f9-11ea-9666-49f50c4cac18.png)

```
Mono JIT compiler version 6.8.0.105 (tarball Tue Feb  4 21:20:20 UTC 2020)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
	TLS:           __thread
	SIGSEGV:       altstack
	Notifications: epoll
	Architecture:  amd64
	Disabled:      none
	Misc:          softdebug 
	Interpreter:   yes
	LLVM:          yes(610)
	Suspend:       hybrid
	GC:            sgen (concurrent by default)
```

**Comment by Sanaki** [comment]
![mesenheaderbuilt](https://user-images.githubusercontent.com/1374333/80912384-23632900-8d2c-11ea-9ed8-8a9429dae372.png)
Just an updated image since moving to self-built. Seems like it should be able to display it all, it just ran out of room. This one at least shows 12 bytes.

**Comment by SourMesen** [comment]
Mono's layout seems to vary from one install to the other, and I haven't been able to figure out why.  

On my Kubuntu 18 VM, the majority of the interface looks fine (because that's how Mono has looked in the past and I've done a lot of adjustments for it), but on Kubuntu 19, with the exact same version of Mono installed, a *lot* of the layout is wrong, with text taking more space than expected, etc.

Until I can figure out a reason for this discrepancy, it's rather hard to fix layout issues properly, unfortunately :\

**Comment by Sanaki** [comment]
Yeah, I didn't really expect a fix any time soon when I opened this issue. Still, I didn't realize it varied -that- much. Rough.
